src/components/CVE-2021-41773/CVE-2021-41773.tsx

```diff
@@ -23,20 +23,39 @@ const description_userguide =
     "       Eg: 2.4.49\n\n" +
     "Step 5: Click Exploit to commence the RCE’s operation.\n\n" +
     "Step 6: View the Output block below to view the results of the attack vectors execution.";
```

Comment:
Vectors is missing an apostrophe. 
```suggestion
    "Step 6: View the Output block below to view the results of the attack vector's execution.";
```